.devcontainer/Dockerfile

```diff
@@ -17,4 +17,4 @@ RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | b
 
 COPY . .
 
-RUN /root/.cargo/bin/uv pip install --no-cache -e .
+RUN /root/.cargo/bin/uv pip install --no-cache -e . --system
```

Comment:
Any reason why?

Reply:
this was already merged in my previous PR; not sure why it's still here but Neil made this fix to help set up the Docker environment

Reply:
yea that shouldn't show up as a diff? odd